src/elements/content-sidebar/activity-feed/task-new/Assignees.js

```diff
@@ -19,20 +20,28 @@ type Props = {|
 |} & InjectIntlProvidedProps;
 
 const statusMessages = {
-    [TASK_NEW_APPROVED]: <FormattedMessage {...messages.tasksFeedStatusAccepted} />,
-    [TASK_NEW_REJECTED]: <FormattedMessage {...messages.tasksFeedStatusRejected} />,
-    [TASK_NEW_COMPLETED]: <FormattedMessage {...messages.tasksFeedStatusCompleted} />,
-    [TASK_NEW_NOT_STARTED]: <FormattedMessage {...messages.tasksFeedStatusNotStarted} />,
+    [TASK_NEW_APPROVED]: messages.tasksFeedStatusAccepted,
+    [TASK_NEW_REJECTED]: messages.tasksFeedStatusRejected,
+    [TASK_NEW_COMPLETED]: messages.tasksFeedStatusCompleted,
+    [TASK_NEW_NOT_STARTED]: null,
 };
 
-const AssigneeTooltipLabel = React.memo(({ user, status }) => {
+const Datestamp = ({ date }: { date: ISODate | Date }) => {
+    return <ReadableTime timestamp={+new Date(date)} alwaysShowTime relativeThreshold={-1} />;
```

Comment:
What the purpose of `+new` here?

Reply:
Cast to number

Reply:
Can we use `.getTime()` or another named method so future developers don't ask the same question?